integration-tests/test/native-eth.spec.ts

```diff
@@ -41,17 +41,17 @@ describe('Native ETH Integration Tests', async () => {
   })
 
   describe('estimateGas', () => {
-    it('Should estimate gas for ETH transfer', async () => {
+    it.only('Should estimate gas for ETH transfer', async () => {
```

Comment:
I have made the mistake of leaving in a `only` as well, I feel like there is a syntax `$ mocha --only` that could prevent these types of problems from happening

Reply:
My bad. Force-pushed should be gone now.